src/dev/jest/mocks/resize_observer_mock.js

```diff
@@ -0,0 +1,43 @@
+/*
```

Comment:
nit: Cert header should be 
```
/*
 * Copyright OpenSearch Contributors
 * SPDX-License-Identifier: Apache-2.0
 */
```
for new created files.

Reply:
Ah, thx, I just copied this from elsewhere - to be clear, should just this comment block be replaced or both comment blocks? Are there docs that explain why these need to be added to each file?

Reply:
Just replaced both